feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -47,17 +42,30 @@ fun RepairView(
         formularyData = {
             repairViewModel.setValues(it)
         },
+        mimeTypes = repairViewModel.mimeTypes,
         validatePhone = { repairViewModel.validatePhone(phone = it) },
         onClickButton = { repairViewModel.onClickSendButton() },
         navigateToApartment = navigateToApartment
     )
 }
 
+/**
+ * Function [RepairContainer] has
```

Comment:
Describe the main pourpose of RepairContainer, for example: contains the view of repair screen... describe it